docs: release notes for 24.7 (#631)
stackabletech/documentation: modules/concepts/pages/multi-platform-support.adoc

= Multi-platform for SDP
:description: This page describes multi-platform support for AMD64 and ARM64
:keywords: Multi-Architecture, infrastructure, docker, image, tags
:multi-platform-images: https://docs.docker.com/build/building/multi-platform/

WARNING: This status is still experimental, as we work to fine-tune the necessary workflows.

== Description

Starting with Stackable release 24.7, all images are {multi-platform-images}[multi-platform images], supporting the AMD64 and ARM64 architectures.
Each product image is built for each platform with a specific tag.

> These tags are then bundled as a manifest list using an automated workflow so that the appropriate image can be transparently selected for the active platform/architecture.

Reviewer: ```suggestion
These tags are then bundled as a manifest list using an automated workflow so that the appropriate image can be transparently selected for the active platform/architecture.
This also enables mixed Kubernetes clusters, with amd64 and arm64 nodes being used simulatiously. 
```

= Multi-platform for SDP
:description: This page describes multi-platform support for AMD64 and ARM64
:keywords: Multi-Architecture, infrastructure, docker, image, tags
:multi-platform-images: https://docs.docker.com/build/building/multi-platform/

WARNING: This status is still experimental, as we work to fine-tune the necessary workflows.

== Description

> Starting with Stackable release 24.7, all images are {multi-platform-images}[multi-platform images], supporting the AMD64 and ARM64 architectures.

Reviewer 2: ```suggestion
Starting with Stackable release 24.7, all images support {multi-platform-images}[multi-platform images] the AMD64 and ARM64 architectures.
```

Reviewer 2: Wait....this is wrong....I think I missed something in the edit.

Reviewer 2: I think you can ignore this suggestion.

Author: Reverted 32ef010

> Each product image is built for each platform with a specific tag.

Reviewer 2: "specific" tag is a bit confusing. What's meant here?

Author: I mean platform-specific e.g. 2.9.2-stackable24.7.0-amd64 and 2.9.2-stackable24.7.0-arm64

Author: i'll make that clearer

Author: See 32ef010

== Description

Starting with Stackable release 24.7, all images are {multi-platform-images}[multi-platform images], supporting the AMD64 and ARM64 architectures.
Each product image is built for each platform with an architecture-specific tag.

> For example, the airflow images with tags `airflow:2.9.2-stackable24.7.0-amd64` and `airflow:2.9.2-stackable24.7.0-arm64` are bundled in the manifest list `airflow:2.9.2-stackable24.7.0` using an automated workflow.

Reviewer 2: ```suggestion
For example, the Airflow images with tags `airflow:2.9.2-stackable24.7.0-amd64` and `airflow:2.9.2-stackable24.7.0-arm64` are bundled in the manifest list `airflow:2.9.2-stackable24.7.0` using an automated workflow.
```
